.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: Build External Images
+
+on:
+  workflow_dispatch:
+    inputs:
+      list_of_repo_branch_tag:
+        description: "Space separated list of repo:branch:tag. For example:csi-attacher:longhorn-release-4.6.1:v4.6.1-20241007 csi-provisioner:longhorn-release-4.0.1:v4.6.1-20241007"
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_ACTION_TOKEN }}
+
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+    - name: Setup Git
+      run: |
+        gh auth setup-git
+
+    - uses: anchore/sbom-action/download-syft@v0.17.2
+
+    - run: |
+        git config --global user.email "derek.su@suse.com"
```

Comment:
Let's try to use the Longhorn GitHub app later for all workflow pipelines including release.

![image](https://github.com/user-attachments/assets/bbce3ee7-ed14-40ad-b2b3-bbb22082f0e6)